packetbeat/docs/fields.asciidoc

```diff
@@ -626,6 +632,284 @@ The time interval in seconds that this resource record may be cached before it s
 The data describing the resource. The meaning of this data depends on the type and class of the resource record.
 
 
+=== amqp Fields
+
+AMQP specific event fields.
+
+
+==== amqp.reply-code
+
+type: int
+
+example: 404
+
+AMQP reply code to an error, similar to http reply-code
+
+
+==== amqp.reply-text
+
+type: string
+
+Text expliciting the error.
```

Comment:
I'll include the suggested edit in a new PR. I didn't write any of this AMQP documentation. The fields.asciidoc hadn't been updating since AMQP was contributed.